maquetado/scss/detail.scss

```diff
@@ -93,3 +95,74 @@
   font-size: 20px;
   line-height: 23px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .body {
+    align-items: center;
+    display: flex;
+    flex-direction: column-reverse;
+  }
+
+  .card {
+    align-items: center;
+    flex-direction: column;
+    height: 570px;
+    margin-top: 100px;
+    padding: 40px 20px 0 20px;
+    width: 335px;
+  }
+
+  .photo-detail {
+    height: 100%;
+    width: 100%;
+  }
+
+  .photo {
+    height: 300px;
+    margin-bottom: 20px;
+    width: 215px;
+
+    &::after {
+      height: calc(#{$badge-height} - #{$badge-height} * 0.3);
+      top: 25px;
+      transform: rotate(15deg);
+      width: calc(#{$badge-width} - #{$badge-width} * 0.3);
+    }
+  }
+
+  .information {
+    width: 100%;
+  }
+
+  .book-description {
+    margin-bottom: 20px;
+  }
+
+  .book-description:first-child {
+    padding-bottom: 10px;
```

Comment:
you can add this style of book-description at line 137 using ampersand character.
```
.class {
  ...
  &:pseudo-class{ }
}
```